controllers/postController.js

```diff
@@ -100,4 +102,159 @@ const list = async (req, res) => {
   }
 };
 
-module.exports = { upsert, list };
+const today = async (req, res) => {
+  if (!isValidString(req.params.keywordId) || isEmptyString(req.params.keywordId)) {
+    return res.status(400).send({ message: "[InvalidKeywordId] Error occured" });
+  }
+
+  const hasKeywordId = (await keywordModel.findOne({ _id: req.params.keywordId }).exec()) !== null;
+  if (!hasKeywordId) {
+    return res.status(400).send({ message: "[NotExistedKeywordId] Error occured" });
+  }
+
+  const keywordId = req.params.keywordId;
+
+  try {
+    const today = new Date();
+    const yesterday = new Date(today.getFullYear(), today.getMonth(), today.getDate() - 1);
+
+    const todayPostCount = await postModel
+      .find({ keywordId })
+      .find({
+        createdAt: {
+          $gte: today.setHours(0, 0, 0, 0),
+          $lt: today.setHours(23, 59, 59, 999),
+        },
+      })
+      .countDocuments()
+      .exec();
+    const yesterdayPostCount = await postModel
+      .find({ keywordId })
+      .find({
+        createdAt: {
+          $gte: yesterday.setHours(0, 0, 0, 0),
+          $lt: yesterday.setHours(23, 59, 59, 999),
```

Comment:
위 코멘트와 동일한 맥락에서 아래와 같이 변경하는 것이 어떨지요?

```suggestion
          $gte: new Date(yesterday).setHours(0, 0, 0, 0),
          $lt: new Date(yesterday).setHours(23, 59, 59, 999),
```

Reply:
해당 사항 수정했습니다!